src/Menubar.tsx

```diff
@@ -435,24 +467,27 @@ function PictureInner({ config, onConfigUpdate, disabled }: PictureProps) {
 
   const updateConfig = useCallback(onConfigUpdate || ((config: Partial<api.Config>) => {}), [
     onConfigUpdate,
-  ]);
+  ]); // set the update of the configuration file
 
   const updateContrast = useDebounceCallback(
+    // update the contrast parameter with a debounce to don't call the api in each change
     (value: number) => updateConfig({ contrast: `${value}` }),
     DEBOUNCE_TIME
   );
 
   const updateSharpness = useDebounceCallback(
+    // update the sharpness parameter with a debounce to don't call the api in each change
```

Comment:
```suggestion
  const updateSharpness = useDebounceCallback(
    // update the sharpness parameter with a debounce to avoid spamming the api for each change
```